src/config.rs

```diff
@@ -94,18 +132,19 @@ impl NetworkConfig {
             faucet_url: None,
             meta_transaction_relayer_url: None,
             fastnear_url: Some("https://api.fastnear.com/".parse().unwrap()),
-            staking_pools_factory_account_id: Some("pool.near".parse().unwrap()),
+            staking_pools_factory_account_id: Some("poolv1.near".parse().unwrap()),
         }
     }
 
+    /// Constructs default testnet configuration.
     pub fn testnet() -> Self {
         Self {
             network_name: "testnet".to_string(),
             rpc_endpoints: vec![RPCEndpoint::testnet()],
             linkdrop_account_id: Some("testnet".parse().unwrap()),
             near_social_db_contract_account_id: Some("v1.social08.testnet".parse().unwrap()),
             faucet_url: Some("https://helper.nearprotocol.com/account".parse().unwrap()),
-            meta_transaction_relayer_url: Some("http://localhost:3030/relay".parse().unwrap()),
+            meta_transaction_relayer_url: None,
```

Comment:
but this crate being a lib, it won't be too great to use a dependency with experimental feature, which can break on minor releases, 

though pinning it with `~3.3.1` (only patch releases updates allowed) or `=3.3.1` (fully pinned) would allow to avoid problems for downstream consumers of the library

Reply:
To be honest, I don't really want ro pull some extra dep for that 

Ideally is to remove, and keep it as slim as possible

Reply:
If you were to use `bon` builder for this, you could alternatively do it this way:

```rust
// A `derive(bon::Builder)` here isn't required, but it's not prohibited!
// If you want to have both `Network::builder()` for regular code and
// `Network::testnet()` for test code, then both can coexist just fine
struct Network {
    name: String,
    rpc_stuff: String,
}

// Define a builder using the method syntax
#[bon::bon]
impl Network {
    #[builder]
    pub fn testnet(
        #[builder(default = "testnet.into())]
        name: String,
        #[builder(default = "my_fast_near_rpc".into())]
        rpc_stuff: String,
    ) -> Self {
        Self { name, rpc_stuff }
    }
}

// Both of these work as expected
Network::testnet().build();
Network::testnet().rpc_stuff("very_slow_another_rpc".to_owned()).build();
```

cc @dj8yfo

Reply:
@akorchyn well, this dependency has a useful function, it's really convenient to configure options with this stuff, i've tried. Besides Veetaha just showed how to do this with stable features of bon. Anyway, this was just a suggestion how stuff could be improved, not how it must be improved imo.
After having near-*-0.28 in, bon wouldn't be too much of additional weight, cannot compile near-workspaces tests --workspace with default number of builders, only small number -j2 works, othewise it hangs and is later killed with oom.